pkg/sysregistriesv2/system_registries_v2_test.go

```diff
@@ -675,6 +675,72 @@ func TestPullSourcesFromReference(t *testing.T) {
 	assert.Equal(t, 1, len(pullSources))
 }
 
+func TestPullSourcesMirrorFromReference(t *testing.T) {
+	sys := &types.SystemContext{
```

Comment:
To make it easier for the future, could `TestPullSourcesFromReference` be modified to use the same table-driven format, and then these tests added to that function, without adding a new config file?

It’s a bit out of scope (especially with the testing for `Location`/`Insecure`), so I’m perfectly fine with doing that work myself later, in a separate PR.

Reply:
ok, I'd leave that to you.